packages/fiori/src/ViewSettingsDialog.hbs

```diff
@@ -0,0 +1,57 @@
+<ui5-dialog
+	aria-label="{{_dialogTitle}}"
+	@after-open={{_focusRecentlyUsedControl}}
+	@before-close={{_restoreConfirmedOnEscape}}
+	?stretch={{_isPhone}}
+>
+
+	<div slot="header" class="ui5-vsd-header">
+		<ui5-bar>
+			<ui5-label
+				slot="startContent"
+				class="ui5-vsd-title">{{_dialogTitle}}</ui5-label>
+			<ui5-button
+				slot="endContent"
+				design="Transparent"
+				@click="{{_resetSettings}}"
+				?disabled={{_disableResetButton}}
+				reset-button>{{_resetButtonLabel}}</ui5-button>
+		</ui5-bar>
+	</div>
+
+	<div class="ui5-vsd-content">
+		<div class="ui5-vsd-sort">
+			<ui5-list
+				mode="SingleSelectBegin"
+				@selection-change="{{_onSortOrderChange}}"
+				sort-order>
+				<ui5-li-groupheader>{{_sortOrderLabel}}</ui5-li-groupheader>
+				<ui5-li
+					?selected="{{_sortAscending}}"
+				>{{_ascendingLabel}}</ui5-li>
+				<ui5-li
+					?selected="{{sortDescending}}"
+				>{{_descendingLabel}}</ui5-li>
+			</ui5-list>
+			<ui5-list
+				mode="SingleSelectBegin"
+				@selection-change={{_onSortByChange}}
+				sort-by>
+				<ui5-li-groupheader>{{_sortByLabel}}</ui5-li-groupheader>
+				<slot name="sorting"></slot>
+			</ui5-list>
+		</div>
+	</div>
+
+	<div slot="footer" class="ui5-vsd-footer">
+		<ui5-button
```

Comment:
I noticed that the confirmation button is not wide enough:
currently it is like that:
<img width="257" alt="Screenshot 2021-07-16 at 15 01 54" src="https://user-images.githubusercontent.com/15702139/125944384-4ba5e679-0452-48d5-ac36-b5b3335d6103.png">

but if you check OpenUI5 https://openui5nightly.hana.ondemand.com/entity/sap.m.ViewSettingsDialog/sample/sap.m.sample.ViewSettingsDialog

<img width="332" alt="Screenshot 2021-07-16 at 15 00 56" src="https://user-images.githubusercontent.com/15702139/125944296-d2bf26da-c31a-400b-ba9d-1dd250d02a8b.png">

The button should have some min-width.

Reply:
done